prow/cmd/jenkins-operator/main.go

```diff
@@ -230,14 +240,6 @@ func main() {
 	}
 }
 
-func loadToken(file string) (string, error) {
-	raw, err := ioutil.ReadFile(file)
-	if err != nil {
-		return "", err
-	}
-	return string(bytes.TrimSpace(raw)), nil
-}
-
 func loadCerts(certFile, keyFile, caCertFile string) (*tls.Config, error) {
 	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
```

Comment:
Open a follow-up issue to add support for x509 certs in the secret agent.

Reply:
opened #8701